### ISSUE TITLE
java.lang.NoClassDefFoundError on Docker

### ISSUE THREAD

**Issue by noxowl** [Nachtbeere/Zen#6]
```
java.lang.NoClassDefFoundError: Could not initialize class org.jetbrains.exposed.dao.EntityCacheKt
	at org.jetbrains.exposed.dao.EntityLifecycleInterceptor.beforeRollback(EntityLifecycleInterceptor.kt:47) ~[?:?]
	at org.jetbrains.exposed.sql.Transaction.rollback(Transaction.kt:72) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.TransactionApiKt.rollbackLoggingException(TransactionApi.kt:106) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt$inTopLevelTransaction$1.invoke(ThreadLocalTransactionManager.kt:191) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt$inTopLevelTransaction$2.invoke(ThreadLocalTransactionManager.kt:211) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.keepAndRestoreTransactionRefAfterRun(ThreadLocalTransactionManager.kt:219) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.inTopLevelTransaction(ThreadLocalTransactionManager.kt:210) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt$transaction$1.invoke(ThreadLocalTransactionManager.kt:148) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.keepAndRestoreTransactionRefAfterRun(ThreadLocalTransactionManager.kt:219) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.transaction(ThreadLocalTransactionManager.kt:120) ~[?:?]
	at org.jetbrains.exposed.sql.transactions.ThreadLocalTransactionManagerKt.transaction(ThreadLocalTransactionManager.kt:118) ~[?:?]
	at net.nachtbeere.minecraft.zen.ZenStorage.initialize(ZenStorage.kt:38) ~[?:?]
	at net.nachtbeere.minecraft.zen.Zen.onLoad(Zen.kt:33) ~[?:?]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.loadPlugins(CraftServer.java:365) ~[patched_1.15.2.jar:git-Paper-391]
	at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:234) ~[patched_1.15.2.jar:git-Paper-391]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:909) ~[patched_1.15.2.jar:git-Paper-391]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_242]
```

**Comment by noxowl** [comment]
Develop environment: Java 1.8 (OpenJDK 64-Bit Server VM 25.242-b08) Host: Mac OS X 10.15.7 (x86_64)
Docker: Java 1.8 (OpenJDK 64-Bit Server VM 25.242-b08) Host: Linux 3.10.0-1062.18.1.el7.x86_64 (amd64)

**Comment by noxowl** [comment]
resolved in v0.5
